spark3/src/main/java/org/apache/iceberg/actions/Spark3MigrateAction.java

```diff
@@ -122,6 +122,8 @@ public Long execute() {
     properties.put(TableCatalog.PROP_PROVIDER, "iceberg");
     properties.put("migrated", "true");
     properties.putAll(additionalProperties());
+    properties.putIfAbsent(LOCATION, sourceTableLocation());
+
```

Comment:
I'd actually add some empty lines to separate logical blocks similar to what you have in snapshot.

```
    Map<String, String> properties = Maps.newHashMap();

    properties.putAll(JavaConverters.mapAsJavaMapConverter(v1SourceTable().properties()).asJava());
    EXCLUDED_PROPERTIES.forEach(properties::remove);

    properties.put(TableCatalog.PROP_PROVIDER, "iceberg");
    properties.put("migrated", "true");
    properties.putAll(additionalProperties());
    properties.putIfAbsent(LOCATION, sourceTableLocation());

    return properties;
```